tricat_211/WallObstacle[] walls
